npm/iptm/iptm.go

```diff
@@ -343,6 +354,17 @@ func (iptMgr *IptablesManager) Save(configFile string) error {
 		configFile = util.IptablesConfigFile
 	}
 
+	l, err := grabIptablesLocks()
+	if err != nil {
+		return err
+	}
+
+	defer func(l *os.File) {
+		if err = l.Close(); err != nil {
+			fmt.Printf("Failed to close iptables locks")
```

Comment:
Did you mean to swallow this error? Should this at least be a warning?

Reply:
@serbrech 
Since it's in 
```
defer{}
```
There's no clean way of handling this. I changed fmt.Printf to log.Printf so telemetry will catch it.